agent/internal/util/json.go

```diff
@@ -7,7 +7,7 @@ import (
 func RemoveJSONComment(strIn []byte) []byte {
 	str := string(strIn)
 	// edgecase that isn't handled: double quote in the comment
-	re1 := regexp.MustCompile(`//\s*?[^"\\S]*?\s*?(\n|\r|\r\n|\n\r)`)
+	re1 := regexp.MustCompile(`([ \t]*)//\s*?[^"\\S]*?\s*?(\n|\r|\r\n|\n\r)`)
```

Comment:
\s includes tabs too.

> That is: \s matches a space, a tab, a carriage return, a line feed, or a form feed.
Source: https://www.regular-expressions.info/shorthand.html

Also, golang official: https://pkg.go.dev/regexp/syntax
```
\s             whitespace (== [\t\n\f\r ])
```

Is it necessary?

Reply:
You are right, `\s` is enough. Corrected the regex.